recipe/meta.yaml

```diff
@@ -1,7 +1,7 @@
 # Make sure the component versions are consistent with the SDK version!
-{% set version = "24.03.0.4" %}
+{% set version = "24.08.0.5" %}
```

Comment:
Why 24.08.0.5?

Reply:
It’s the apparent version from the tar.gz on DevZone.